etl/data_extraction/Scraper.py

```diff
@@ -120,13 +120,28 @@ def add_urls(self):
         self.urls.append(self.base_url)
 
     @staticmethod
-    def clean_string(value):
-        """Removes all line breaks, tabs and leading or trailing whitespaces from a string."""
-
+    def remove_unnecessary_whitespaces(value):
+        """Removes duplicate and leading or trailing whitespaces in a string"""
         if value is None:
             return None
+        return ' '.join(value.split())
+
+    @staticmethod
+    def clean_string(value, *optional_replacement_argument):
+        """Removes all line breaks, tabs and leading or trailing whitespaces from a string. The optional argument
+        optional_replacement_argument lets you specify a string to substitute the replaced tags"""
 
-        return value.replace('\n', '').replace('\r', '').replace('\t', '').strip()
+        if value is None:
+            return None
+        if len(optional_replacement_argument) is 0:
```

Comment:
Please replace 'is' with '=='.

Reply:
implemented requested change. I was planning to create the frontend issue after the merge